README.md

```diff
@@ -1,2 +1,79 @@
-# devfile-sample-java-springboot-basic
-A basic sample application using Java Spring Boot with devfile
+# sample-devfile-with-basic-java-spring-boot
+
+By default, the Spring Boot application uses the **HTTP 8080** port. You can use a devfile to define how Spring Boot interacts with your application.
+
+## Running Spring Boot from the command line
+
+You can run Spring Boot with the `java -jar` command, like this:
+
+:source-highlighter: highlightjs
+:source-language: java
+
+[source]
+----
+$ java -jar target/myapplication-0.0.1-SNAPSHOT.jar
+----
+
+## Example devfile
```

Comment:
Lets not mention the entire devfile in the README. Maybe we can mention the file `devfile.yaml` and perhaps direct them to devfile.io for more information if they're curious

Reply:
thanks! what do we say about the .yaml? where is it?

Reply:
it is here https://github.com/devfile-samples/devfile-sample-java-springboot-basic/blob/main/devfile.yaml

I think we can say this file defines the lifecycle of the workspace. The outerloop phase of the application lifecycle is covered by the highlighted lines https://github.com/devfile-samples/devfile-sample-java-springboot-basic/blob/main/devfile.yaml#L21-L41

Reply:
thank you, @maysunfaisal. what do you mean by "the outerloop phase"?